Revit_Core_Engine/Query/Bounds.cs

```
@@ -81,6 +81,27 @@ public static BoundingBoxXYZ Bounds(this IEnumerable<Solid> solids, Transform tr
         }
 
         /***************************************************/
+
+        [Description("Returns the combined outline of the given outlines.")]
+        [Input("outlines", "A list of outlines to combine.")]
+        [Output("bounds", "Combined outline of the given outlines.")]
+        public static Outline Bounds(this List<Outline> outlines)
+        {
+            if (outlines == null || !outlines.Any())
+                return null;
```

I would throw an error about null or empty collection here

---

Done 👍